src/DroneSafetyApi/Models/HeatmapsResponse.cs

```diff
@@ -2,10 +2,22 @@
 
 namespace DroneSafetyApi.Models
 {
+    /// <summary>
+    /// The HeatmapResponse class is a model to hold information that is being sent to the web application
```

Comment:
We shouldn't mention the web application. Would be better to say, "model to hold information served over the api"